pkg/commands/set_cluster_test.go

```diff
@@ -32,9 +37,22 @@ func TestSetClusterFields(t *testing.T) {
 		{
 			Fields: []string{
 				"spec.kubernetesVersion=1.8.2",
+				"spec.kubelet.authorizationMode=Webhook",
+				"spec.kubelet.authenticationTokenWebhook=true",
+			},
+			Input: kops.Cluster{
+				Spec: kops.ClusterSpec{
+					Kubelet: &kops.KubeletConfigSpec{},
+				},
 			},
 			Output: kops.Cluster{
-				Spec: kops.ClusterSpec{KubernetesVersion: "1.8.2"},
+				Spec: kops.ClusterSpec{
+					KubernetesVersion: "1.8.2",
+					Kubelet: &kops.KubeletConfigSpec{
+						AuthorizationMode:          "Webhook",
+						AuthenticationTokenWebhook: truePointer(),
```

Comment:
Tip for next time: You could use `fi.Bool(true)` or `aws.Bool(true)` instead of truePointer